x-pack/plugins/index_management/public/lib/ace.js

```diff
@@ -55,7 +55,7 @@ export const createAceEditor = (
   session.setUseWrapMode(true);
   session.setMode("brace/mode/json");
```

Comment:
I assume this needs to be updated too? I'm seeing a 404 in the devtools for `http://localhost:5601/qrl/app/mode-json.js` and the editor doesn't have JSON highlighting.